### PR TITLE
Narven/typo fix

### PR DESCRIPTION
**Pull request by NaraVen**
typo fix and one manual non-automated test file

### CODE REVIEW

File: test/not-automated/multiprocessing-threading/multithreading.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from threading import Thread
+import os
+
+def threaded_function():
+    top = "/home/nara/mntblobfuse/128/wiki_pretrain/"
```
Comment (vibhansa-msft) [inline test/not-automated/multiprocessing-threading/multithreading.py:5]: We want to make this script generic and add it to the pipeline ? I see the paths are hard-coded here.

File: blobfuse/src/BlobfuseGcCache.cpp [hunk 2]
```diff
@@ -205,6 +205,10 @@ void gc_cache::run_gc_cache()
                             // Failed to acquire the lock for some other reason.  We close the open fd, and continue.
                             syslog(LOG_ERR, "Did not clean up file %s from file cache because we failed to acquire the flock for an unknown reason, errno = %d.\n", mntPath, errno);
                         }
+                        if (permissionsoverwritten == true)
+                        {
+                            fchmod(fd, originalpermissions);
```
Comment (vibhansa-msft) [inline blobfuse/src/BlobfuseGcCache.cpp:210]: We shall also reset the permission when open call fails (around line 236)

Comment (NaraVen) [reply blobfuse/src/BlobfuseGcCache.cpp:210]: done

**Review by vibhansa-msft — APPROVED** [review]

File: blobfuse/src/BlobfuseGcCache.cpp (hunk 2 above)
Comment (vibhansa-msft) [inline blobfuse/src/BlobfuseGcCache.cpp:210]: There is build failure due to this statement :

/home/vsts/work/1/s/blobfuse/src/BlobfuseGcCache.cpp:210:35: error: ‘originalpermissions’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
                             fchmod(fd, originalpermissions);

we need to initialize this variable in the begining.

**Pull request merged**